content/en/serverless/installation/nodejs.md

```diff
@@ -96,21 +97,17 @@ The [Datadog Serverless Plugin][1] automatically adds the Datadog Lambda Library
 To install and configure the Datadog Serverless Plugin, follow these steps:
 
 1. Install the Datadog Serverless Plugin:
-	  ```
+    ```
     yarn add --dev serverless-plugin-datadog
     ```
 2. In your `serverless.yml`, add the following:
-    ```
-    plugins:
-      - serverless-plugin-datadog
-    ```
-<div class="alert alert-info">If you are instead deploying your Serverless Framework app <a href="https://www.serverless.com/framework/docs/providers/aws/guide/intro">by natively exporting a JSON object from a Javascript file</a> (for example, by using a <code>serverless.ts</code> file), follow the <a href="https://docs.datadoghq.com/serverless/installation/nodejs?tab=custom">custom installation instructions</a>.</div>
+`plugins: - serverless-plugin-datadog`
```

Comment:
Is there a reason this is compressed onto one line? I'm not sure if this is valid YAML